activemodel/lib/active_model/errors.rb

```diff
@@ -613,13 +636,27 @@ def add_from_legacy_details_hash(details)
         }
       end
 
-      def deprecation_removal_warning(method_name)
-        ActiveSupport::Deprecation.warn("ActiveModel::Errors##{method_name} is deprecated and will be removed in Rails 6.2")
+      def deprecation_removal_warning(method_name, alternative_message = nil)
+        message = +"ActiveModel::Errors##{method_name} is deprecated and will be removed in Rails 6.2."
+        if alternative_message
+          message << "\n\nTo achieve the same use:\n\n  "
+          message << alternative_message
+        end
+        ActiveSupport::Deprecation.warn(message)
       end
 
       def deprecation_rename_warning(old_method_name, new_method_name)
         ActiveSupport::Deprecation.warn("ActiveModel::Errors##{old_method_name} is deprecated. Please call ##{new_method_name} instead.")
       end
+
+      def deprecation_index_access_warning(method_name, alternative_message)
+        message = +"ActiveModel::Errors##{method_name} is deprecated. In the next release it would return `Error` object instead."
```

Comment:
ditto
```suggestion
        message = "ActiveModel::Errors##{method_name} is deprecated. In the next release it would return `Error` object instead."
```

Reply:
Same as above, I need to manipulate the string later.